contracts/pods/PodFactory.sol

```diff
@@ -267,43 +259,36 @@ contract PodFactory is CoreRef, IPodFactory {
         return podController.podIdToSafe(podId);
     }
 
-    /// @notice Create an optimistic timelock, linking to an Orca pod
+    /// @notice Create a timelock, linking to an Orca pod
     /// @param safeAddress Address of the Gnosis Safe
     /// @param minDelay Delay on the timelock
     /// @param publicExecutor Non-permissioned smart contract that
     ///        allows any address to execute a ready transaction
-    /// @param vetoController Address which manages veto rights over a pod timelock
+    /// @param podAdmin Address which is the admin of the Orca pods
     ///        Will also be able to propose
-    // TODO: Prefix with _
-    function createOptimisticTimelock(
+    function _createTimelock(
         address safeAddress,
         uint256 minDelay,
         address publicExecutor,
         address podAdmin
     ) internal returns (address) {
-        // TODO: Validate minDelay
-        address[] memory proposers = new address[](2);  // cancel timelock
+        address[] memory proposers = new address[](2);
         proposers[0] = safeAddress;
-        // Note: If you migrate the podAdmin, then it will not have the timelock propsoer role. So it
-        // will not be able to cancel the timelock.
-        // TODO: Remove podAdminGateway
         proposers[1] = podAdmin;
 
         address[] memory executors = new address[](2);
         executors[0] = safeAddress;
         executors[1] = publicExecutor;
 
-        // TODODODODODO: Use TimelockController from OZ
-        OptimisticTimelock timelock = new TimelockController(
-            address(core()),
+        TimelockController timelock = new TimelockController(
```

Comment:
we should use v4.6 when it releases https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v4.6/contracts/governance/TimelockController.sol and be more specific with the roles

Reply:
Agreed. Didn't want to pull the contract direct from Github pre-release. Reaching out to ask about v4.6.0 release timelines

Reply:
Still looks to be 4.5. Can we import the more recent commits via npm override?

Reply:
Going to update now in a seperate PR, merging this down

Reply:
Upgraded here: https://github.com/fei-protocol/fei-protocol-core/pull/739